src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +15,63 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
```

Comment:
```suggestion
    public Comment create(Comment comment) {
```
let's be more specific (same for other methods)

Reply:
done